seed/static/seed/js/seed.js

```diff
@@ -168,7 +168,7 @@ SEED_app.run([
     make('release:use_pint', 'boolean', false);
 
     var make2 = _.partial(flippers.make_flipper, 'nicholas.long@nrel.gov', '2018-01-01T00:00:00Z');
-    make2('release:bricr', 'boolean', false);
+    make2('release:bricr', 'boolean', true);
```

Comment:
Ooooh adding another flipper?

Reply:
i am just enabling the flipper on the bricr branch. when we merge down, we will need to disable again.